ImageD11/sinograms/dataset.py

```diff
@@ -418,6 +429,10 @@ def import_motors_from_master(self):
                 print(
                     "replace bad scan omega", b, self.scans[b], "with", j, self.scans[j]
                 )
+        # ensure dty is in microns - this is for compatability with geometry
+        if dty_is_mm:
+            self.dty = self.dty * 1000
```

Comment:
To modify the dty numbers is a breaking change. It should not go into the next release.